tasks/publisher.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: publisher
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Include Apache tasks
+  ansible.builtin.include_tasks: apache.yml
+
+- name: Create a directory cvmfs_keys
+  ansible.builtin.file:
+    path: /tmp/keys
```

Comment:
Is it expected to install the key in `/tmp/`?

Reply:
I followed the document recommendation.
The three key files for the repository (.pub, .crt, and .gw) have been copied from the gateway machine onto the publisher machine, in the directory /tmp/test.cern.ch_keys/.